src/carousel/carousel.spec.ts

```diff
@@ -882,6 +885,186 @@ describe('ngb-carousel', () => {
 
 });
 
+if (isBrowserVisible('ngb-carousel animations')) {
+  describe('ngb-carousel animations', () => {
+
+    @Component({
+      template: `
+      <ngb-carousel (slid)="onSlid($event)" [interval]="-1">
+        <ng-template ngbSlide id="one">One</ng-template>
```

Comment:
Just an idea, but don't we want to have `(slid)` at this level as well? as in other components?